manifests/d/DygmaLabs/Bazecor/0.3.3.271/DygmaLabs.Bazecor.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: DygmaLabs.Bazecor
+PackageVersion: 0.3.3.271
+PackageLocale: en-US
+Publisher: DygmaLab SL
```

```suggestion
Publisher: Dygma Lab S.L.
```
This must match Control Panel.